backend/apps/slack/tests/test_slack_sync_data.py

```
@@ -0,0 +1,128 @@
+            {
+                "ok": True,
+                "channels": [
+                    {"id": "C1", "name": "general", "num_members": 5},
+                    {"id": "C2", "name": "random", "num_members": 3}
+                ],
+                "response_metadata": {"next_cursor": ""}
+            }
+        ]
+
+
+    def tearDown(self):
```

Ruff (0.11.9)

Redefinition of unused `tearDown` from line 38

(F811)